test/flows/check_uk_visa_flow_test.rb

```diff
@@ -18,7 +18,8 @@ class CheckUkVisaFlowTest < ActiveSupport::TestCase
     @b1_b2_country = "syria"
     @youth_mobility_scheme_country = "canada"
 
-    @eta_text = "If you’re travelling on or after 22 February 2024, you’ll need to apply for an electronic travel authorisation (ETA)"
```

Comment:
was this a redundant text that we just removed?

Reply:
No—it was split into two separate pieces of text (one for non-visa nationals, and one for EEA country nationals). See the next two added lines.